tests/features/fixtures/minimalapp/app/bugsnag.template

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.application'
+apply plugin: 'com.bugsnag.android.gradle'
```

Comment:
It might be possible to depend on bugsnag conditionally by passing a command line argument when making the build. this would avoid the scenario where the dependency versions of something like `com.android.support:appcompat` gets out of sync between the two versions.

  https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_task_using_project_properties